src/rgw/rgw_log.cc

```diff
@@ -410,8 +409,8 @@ void OpsLogFile::start() {
 
 void OpsLogFile::stop() {
   {
-    std::unique_lock lock(log_mutex);
-    cond_flush.notify_one();
+    std::unique_lock lock(mutex);
+    cond.notify_one();
```

Comment:
should stopped be set before the cond.notify_one()?

Reply:
Looks like it should be fine since the mutex is still held while stopped is set